frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -139,9 +139,10 @@ const MessageInput = (props: Props) => {
   }, [selectedTemplate]);
 
   const sendMessage = () => {
-    if (!channelConnected) {
+    if ((!selectedTemplate && !selectedSuggestedReply && !input) || !channelConnected) {
       return;
```

Comment:
we return when the user clicks on the send button and there isn't anything to be send (i.e. when the input text, the template selector and suggestedReply are empty)

Reply:
There is also already this [line](https://github.com/airyhq/airy/blob/b00bf27e959d4b6304f2c9e787e9ee09a1de4966/frontend/ui/src/pages/Inbox/MessageInput/index.tsx#L354). Perhaps we could turn it into a function `canSendMessage` and then re-use it in both places? 🤔

Reply:
good catch 👍 fixing now!